Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ however, it has to be formatted properly to pass verification tests.
 ### Added
 - Preliminary support for visionOS.
 - Show a list of `Derived Bindings` underneath the Binding Path editor to show all controls that matched.
-
+- Support for Game rotation vector on Android
```

Comment:
This new changelog should be moved below `##[Unreleased]` as 1.7.0 was already released. 
Thanks for the PR though 👍 !

Reply:
Also for clarity, maybe mention that this refers to a Sensor

Reply:
This is for fixing a bug. Should I move it to **Fixed** category?

Reply:
I think 'Added' is fine.